bibtexparser/bparser.py

```diff
@@ -142,6 +148,12 @@ def parse(self, bibtex_str, partial=False):
         :return: bibliographic database
         :rtype: BibDatabase
         """
+
+        self._parse_call_count += 1
+        call_counter = self._parse_call_count
+        if call_counter > 1 and not self.expect_multiple_parse:
+            warnings.warn("parser has been called more than once")
```

Comment:
Improve the printed warning: 
- Choose the warning level as one above
- UserWarning type
- Explain how to avoid the warning (setting property `expect_multiple_parse` to true)
- Fix capitaliation

Reply:
```
self._parse_call_count += 1
call_counter = self._parse_call_count
if call_counter == 1 and not self.expect_multiple_parse:
    warnings.warn("Parser has been called more than once, "
                  "avoid the warning by setting property expect_multiple_parse to true", category=UserWarning,
                  stacklevel=2)
```

Would it be ok?

Reply:
Have not tested it yet, but yes, that looks about right. 👍

Reply:
One point I forgot to mention previously: This assignment to a local variable appears not to be needed: `call_counter = self._parse_call_count`. Just use `self._parse_call_count` in the comparison...